### PR TITLE
Ninja updates

### PR DESCRIPTION
**Pull request by ToGWtF**
Refer to issue #1101
### Intent of your Pull Request

As it is now, ninja adrenaline ability simply removes your current stun but does not prevent further stuns. All it takes is a second tase to take you back down.  Using the adrenaline implant more than once gives you ~50 rad damage, which will put you unconscious after the better part of a minute and curing the rad damage is near impossible as you can't pop a pill or use syringes on yourself. Using all 3 adrenaline charges will kill you, because rad damage heals so slowly.

Todo: ~~Changelog~~  ,  ~~Remove debilitating radium penalty from ninja~~  ,  ~~debate change ninja adrenaline to Traitor adrenaline to give a further stun reduction~~ ,  

:cl:
rscadd: "The Spider Clan has finally updated a crucial component of their agent's suits, the Adrenaline injector"
tweak: "Ninja adrenaline no longer has a penalty and works more like a traitor implant, adding a very small period of stun resistance ontop of the immediate effect"
rscadd: "Adds a counter for the remaining charges of your adrenaline injector"
rscadd: "Adds a much needed user message for the adrenal injector to prevent over-use" 
rscadd: "Adds a cooldown for the adrenaline booster"
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
> unironically buffing the most powerful antag short of wizard or blob to give him infinite stun resist with no penalty

you fucking dipshit

**Comment by ToGWtF** [comment]
> buffing the rarest form of antag that hasnt been updated in 3 years and 60% of the playerbase has 0 experience playing and has the hardest objectives

im dropping it from 10u stimulants to 1u though

**Comment by ShadowDeath6** [comment]
Ninja is better than wizard.

**Comment by FluffySurvivor** [comment]
ayyy let's make the invisible built in emag 40 dmg+armor pen 100% embed shitload of dmg stars built in stun **uncounterable in every shape and form**

File: code/modules/ninja/suit/n_suit_verbs/ninja_adrenaline.dm [hunk 1]
```diff
@@ -13,16 +13,23 @@
 		H.SetParalysis(0)
 		H.SetStunned(0)
 		H.SetWeakened(0)
-
+		H.reagents.add_reagent("synaptizine", 1) // Adds roughly 5 seconds of stun-resistance to prevent being re-stunned
+		H.reagents.add_reagent("omnizine", 10)
```
Comment (FluffySurvivor) [inline code/modules/ninja/suit/n_suit_verbs/ninja_adrenaline.dm]: no omnizine.
AT ALL

**Comment by DavidEdwards** [comment]
I don't think the Ninja really needs buffing. Seems most people here agree. Leaving this open for a little while for any positive counter comments. Otherwise this one will be closed.

**Comment by ToGWtF** [comment]
The buff is only VERY slight, though I will remove the omnizine. The update was to remove the fact that one of the ninjas skills kills the ninja if used more than once in a span of 10+ minutes

I don't think players who are notoriously good at this game should be how we balance game modes. I have never seen a ninja greentext, no matter how much keek/bekons/fluffy will claim they will do so easily. The RAREST antag should yes, be the most powerful and should be fun for a new player if they magically roll it.

**Comment by FluffySurvivor** [comment]
I agree with removing the drawback effect of ninja adrenals.
However, I disagree with lingering stun resistance and spamability.
Give it a relatively high cost, remove the stim (this gives move speed, even 1u can get you away from danger pretty fast) and the synaptizine.

**Comment by ToGWtF** [comment]
It has 0 cost, because it only has 3 uses. I can give it a cooldown for using it, so you can use all 3 at once and have almost 20 seconds of stun-resistance.

**Comment by Kmc2000** [comment]
I rolled ninja once

I had 4 minutes to greentext

I did all but one debrain in that time, and was still alive

Ninja is op as shit, doesnt need a buff

**Comment by oisin100** [comment]
I don't think Ninja needs buffing.

**Pull request closed without merging**